doc/installing/macos.md

```diff
@@ -34,6 +34,17 @@ chmod +x openquake-setup-macos-2.1.0-1.run
 ```
 then follow the wizard on screen. By default the code is installed in `~/openquake`
 
+### Upgrade from a previous installation
+
+To upgrade from a previous installation you need to manually remove it first
+
+```bash
+# default is ~/openquake
+rm -Rf /path/to/previous/installation
+```
+Different versions can be installed alongside if a unique destination path is provided.
+
```

Comment:
I would "hide" this information. It is not a good idea to install multiple versions anyway, let's not suggest that this is easy to do.